diff --git a/libthr/thread/thr_mutex.c b/libthr/thread/thr_mutex.c
index cff6451..f19f679 100644
--- a/libthr/thread/thr_mutex.c
+++ b/libthr/thread/thr_mutex.c
@@ -99,7 +99,7 @@ static int	mutex_lock_sleep(struct pthread *, pthread_mutex_t,
 __weak_reference(__pthread_mutex_init, pthread_mutex_init);
 __strong_reference(__pthread_mutex_init, _pthread_mutex_init);
 __weak_reference(__pthread_mutex_lock, pthread_mutex_lock);
-__strong_reference(__pthread_mutex_lock, _pthread_mutex_lock);
+__weak_reference(__pthread_mutex_lock, _pthread_mutex_lock);
 __weak_reference(__pthread_mutex_timedlock, pthread_mutex_timedlock);
 __strong_reference(__pthread_mutex_timedlock, _pthread_mutex_timedlock);
 __weak_reference(__pthread_mutex_trylock, pthread_mutex_trylock);
